docs/UserGuide/Ecosystem-Integration/NiFi-IoTDB.md

```diff
@@ -46,29 +46,30 @@ Apache NiFi includes the following capabilities:
+| Password      | Password to access the IoTDB.                                                                                                                                                                                                                                                                                 | null          | true      |
+| Prefix        | The Prefix begin with root. that will be add to the tsName in data.  <br /> It can be updated by expression language.                                                                                                                                                                                                | null          | true      |
+| Record Reader | Specifies the type of Record Reader controller service to use <br />for parsing the incoming data and determining the schema.                                                                                                                                                                                 | null          | true      |
 | Schema        | The schema that IoTDB needs doesn't support good by NiFi.<br/>Therefore, you can define the schema here. <br />Besides, you can set encoding type and compression type by this method.<br />If you don't set this property, the inferred schema will be used.<br /> It can be updated by expression language. | null          | false     |
-| Aligned       | Whether using aligned interface?  It can be updated by expression language. | false         | false     |
-| MaxRowNumber  | Specifies the max row number of each tablet.  It can be updated by expression language. | 1024          | false     |
+| Aligned       | Whether using aligned interface?  It can be updated by expression language.                                                                                                                                                                                                                                   | false         | false     |
+| MaxRowNumber  | Specifies the max row number of each tablet.  It can be updated by expression language.                                                                                                                                                                                                                       | 1024          | false     |
 
 ### Inferred Schema of Flowfile
 
 There are a couple of rules about flowfile:
 
 1. The flowfile can be read by `Record Reader`.
-2. The schema of flowfile must contains a field `Time`, and it must be the first.
+2. The schema of flowfile must contain a field `Time`, and it must be the first.
 3. The data type of time must be `STRING` or `LONG`.
 4. Fields excepted time must start with `root.`.
```

Comment:
It seems like that the fields do not need to start with `root.`, if the `prefix` property is assigned. Besides, the relationship between `tsName` and field name is not  clarified.

Reply:
Resolved